roles/cinder-common/tasks/ceph_integration.yml

```diff
@@ -1,4 +1,18 @@
 ---
+- name: fetch rados.py
+  get_url: url=https://raw.githubusercontent.com/ceph/ceph/v0.94.3/src/pybind/rados.py
```

Comment:
Might want to make the version of this to grab (`v0.94.3`) a default for the role. It keeps versions together in a similar place.